src/components/SearchModal/styleds.tsx

```diff
@@ -42,7 +42,7 @@ export const TokenPickerItem = styled(Flex)`
   height: 56px;
   cursor: ${({ disabled }) => !disabled && 'pointer'};
   pointer-events: ${({ disabled }) => disabled && 'none'};
-  transition: background-color 0.2s ease;
+  transition: background-color 0.3s ease;
```

Comment:
@luzzif I wasn't sure what you were reffering to about transiton I assumed it was here?